provisioner/group_vars/all.yml

```diff
@@ -10,6 +10,7 @@ valid_workshop_types:
   - networking
   - f5
   - rhel
+  - security_automation
```

Comment:
Small change, but I'd suggest to remove the additional "automation" - all workshops are about automation, so to my understanding the keyword was just differentiating the different use cases.

Reply:
works for me